### ISSUE TITLE
Failure to import any Mealmaster files 

### ISSUE THREAD

**Issue by themaster567** [vabene1111/recipes#886]
### Version
Version: 0.16.8

### Bug description
I have a massive 20 GB (That's when compressed!) collection of recipes and I've been trying to selectively import ones that sounded nice. I'm starting with mealmaster files because those are what I've been seeing the most of besides HTML files and PDFs. The problem is that none of them are importing, Tandoor just asks me to check if I'm importing a valid file. Mealmaster version numbers vary, but so far I've tried ones ranging from version 6 to version 8 with no success. Here are some of the files that I've tried:
[New folder.zip](https://github.com/vabene1111/recipes/files/7147754/New.folder.zip)

**Comment by vabene1111** [comment]
Thank you for the feedback.

Meal master is a little hard to parse due to varying formats, I will try to make the appropriate fixes so that your files can be imported, might take a few weeks though.

**Comment by themaster567** [comment]
> Thank you for the feedback.
> 
> Meal master is a little hard to parse due to varying formats, I will try to make the appropriate fixes so that your files can be imported, might take a few weeks though.

Excellent! That's more than I could have hoped for. I can provide many more samples if you need any.

**Comment by vabene1111** [comment]
ok that is good to know, working on the importer is always easier with samples. If you need help with the other formats let me know, there is a way to import html but not a very automated one but i could think of some features to make that easier

**Comment by vabene1111** [comment]
ok so the first problem was that your files have the `.mmf` extension and i was only looking for `.txt`. This is fixed with the next release.

I also noticed that the import is very slow in my test environment which does not make sense, even though your file contains quite a lot of recipes they should not be so slow to import, investigating this now

**Comment by vabene1111** [comment]
ok the slow speeds seem to be an issue that only occurs on sqlite databases (which is use for testing but are not recommended for production) so this should not be a problem. 

You can simply use your current version, changing the file Extension of your files to `.txt` or await the next release which supports `.mmf`. I have tested some recipes and while not importing perfectly they look usable, if you find certain mealmaster patterns which are not imported well let me know and i can try to improve them.

**Comment by vabene1111** [comment]
that is interesting, i did not test with txt but i think i just changed the filter. But there are quite a few commits since the last release, maybe some fix somewhere else fixed this as well, so i think you will have to wait for the next release (or beta if you are feeling lucky). 

I might release a beta tomorrow if i dont forget it but keep in mind, its use at your own risk and i often forget to udpate it when i update master :D

**Comment by themaster567** [comment]
Understood. I can wait!
